locales/es.js

```diff
@@ -0,0 +1,155 @@
+      sub_title: 'Ahora estamos listos para verificar su identidad',
+      selfie_uploaded: 'Selfie cargado',
+      action: 'Enviar verificación',
+      multiple_docs_uploaded:'Documentos cargados',
+      one_doc_uploaded: 'Documento cargado',
+    },
+    phone_number_placeholder: 'Ingrese el número de celular',
+    loading: 'Cargando...',
+    mobile_connected: {
+      title: {
+        message: 'Conectado a su móvil',
+        submessage: 'Una vez que haya terminado, lo llevaremos al próximo paso',
+      },
+      tips: {
+        item_1: 'Mantenga esta ventana abierta mientras usa su dispositivo móvil',
+        item_2: 'Su enlace móvil caducará en una hora',
+        item_3: 'No actualizar esta página',
+      }
+    },
+    mobile_notification_sent: {
+      title: 'Controle su dispositivo móvil',
+      submessage: 'Hemos enviado un enlace seguro a %{number}',
+      bold_message: 'Puede tardar unos minutos en llegar',
+      tips: {
+        item_1: 'Mantenga esta ventana abierta mientras usa su dispositivo móvil',
+        item_2: 'Su enlace móvil caducará en una hora',
+      },
+      resend_link: 'Reenviar enlace',
+    },
+    switch_device: {
+      header: '¿Necesita usar su móvil para tomar fotos?',
+      submessage: 'Continúe la verificación de forma segura en su dispositivo móvil',
+    },
+    tips: 'Tips',
+  },
+  errors: {
+    invalid_capture: {message:'No se detectó ningún documento', instruction: 'Asegúrese de que todo el documento esté en la foto'},
+    invalid_type: {message: 'Archivo no cargado', instruction: 'Intenta usar otro tipo de archivo'},
+    unsupported_file: {message: 'Tipo de archivo no soportado', instruction: 'Intente usar un archivo .jpg o .png'},
+    invalid_size: {message: 'Tamaño de archivo demasiado grande', instruction: 'El tamaño debe ser menor de 10 MB'},
+    no_face: {message: 'Rostro no encontrado', instruction: 'Su cara debe de estar en la selfie'},
+    multiple_faces: {message: 'Múltiples caras encontradas', instruction: 'Solo su cara puede estar en la selfie'},
+    server_error: {message: 'Conexión perdida', instruction: 'Inténtalo de nuevo'},
+    glare_detected: {message: 'Detectado brillo', instruction: 'Todos los detalles deben ser claros y legibles'},
+    sms_failed: {message: 'Algo salió mal', instruction: "Copie el enlace a continuación en su dispositivo móvil"},
+    sms_overuse: {message: 'Demasiados intentos de reenvío', instruction: "Copie el enlace a continuación en su dispositivo móvil"},
+    lazy_loading: {message: 'Se produjo un error al cargar el componente'},
+    invalid_number: {message: 'Comprueba que su número de móvil sea correcto'},
+    generic_client_error: {message: 'Algo salió mal', instruction: 'Deberá reiniciar tu verificación en su computadora'},
+  },
+  passport: 'Pasaporte',
+  driving_licence: 'Licencia de conducir',
+  national_identity_card: 'Tarjeta de identificación',
+  short_passport: 'Pasaporte',
```

Comment:
the other short_ ones are lower case.

Reply:
Good point, they are supposed to be lower case.